### PR TITLE
Updated / Deleted some examples

### PR DESCRIPTION
**Pull request by ThomasDelteil**
## Description ##

example/multivariate_time_series:
- updated README to point to Gluon implementation
- fixed typo

example/named_entity_recognition:
- fix example and remove need for `preprocessed_data ` data folder

example/nce-loss:
- fix README.md instructions

example/numpy-ops:
- removed outdate API example (NumpyOp)

example/onnx:
- deleted example, duplicate of the tutorial

example/python-howto:
- deleted outdated example

example/mxnet_adversarial_vae:
- renamed to example/vae-gan

### CODE REVIEW

File: example/onnx/super_resolution.py [hunk 1]
```diff
@@ -1,86 +0,0 @@
...
-import mxnet as mx
-from mxnet.test_utils import download
-import mxnet.contrib.onnx as onnx_mxnet
-
-# set up logger
-logging.basicConfig()
-LOGGER = logging.getLogger()
-LOGGER.setLevel(logging.INFO)
-
-def import_onnx():
```
Comment (anirudhacharya) [inline example/onnx/super_resolution.py]: this example is used in one of the ONNX tests - https://github.com/apache/incubator-mxnet/blob/master/tests/python-pytest/onnx/import/onnx_import_test.py#L152 If you are deleting this, you will have to remove the test too.

Comment (ThomasDelteil) [reply example/onnx/super_resolution.py]: The framework tests shouldn't depend on the examples. I will remove that test.

The nightly tests should run the tutorials test and raise an error if the onnx tutorial cannot run

**Comment by ThomasDelteil** [comment]
@anirudh2290 I don't believe there is a need to replace the Monitor ones with Gluon equivalent, as a print() statement is enough to print values in Gluon. Also Monitor seems deprecated since it doesn't appear in the docs and I haven't seen it used in any other examples.

The grouped example is very bare and does not bring anything more than the documentation is already bringing. https://mxnet.incubator.apache.org/versions/1.3.0/api/python/symbol/symbol.html?highlight=group#mxnet.symbol.Group

The image record iter is better served through the docs as well https://mxnet.incubator.apache.org/versions/1.3.0/api/python/io/io.html?highlight=imagerecorditer#mxnet.io.ImageRecordIter

File: example/onnx/super_resolution.py (hunk 1 above)
Comment (anirudhacharya) [reply example/onnx/super_resolution.py]: Agree, having a test for the correctness of this example was, I think, a review comment at the time. Thanks for removing it.

Comment (ThomasDelteil) [reply example/onnx/super_resolution.py]: updated, thanks for pointing it out @anirudhacharya

Comment (marcoabreu) [reply example/onnx/super_resolution.py]: That review comment was from my side. The idea was to introduce test that validate the examples. As of right now, almost all of our examples go entirely untested. But fine with me.

**Comment by roywei** [comment]
@ThomasDelteil Thanks for the contribution!
@mxnet-label-bot [pr-awaiting-merge,Example]

**Comment by ankkhedia** [comment]
@anirudhacharya @anirudh2290 Could you please take a look?

**Comment by anirudhacharya** [comment]
LGTM

**Comment by thomelane** [comment]
LGTM

**Comment by ThomasDelteil** [comment]
@indhub @szha @nswamy can you please merge? Thanks

**Comment by lupesko** [comment]
@sandeep-krishnamurthy can you please help with a review and merge? Thanks!

**Pull request merged**